ui/components/multichain/global-menu/global-menu.js

```diff
@@ -104,7 +107,18 @@ export const GlobalMenu = ({ closeMenu, anchorElement }) => {
   ///: END:ONLY_INCLUDE_IF
 
   return (
-    <Menu anchorElement={anchorElement} onHide={closeMenu}>
+    <Popover
+      referenceElement={anchorElement}
+      isOpen={isOpen}
+      padding={0}
+      onClickOutside={closeMenu}
```

Comment:
Should we also add escape key functionality here? It won't matter for Popup view as pressing esc key will close the extension but in expanded view it would be helpful
```suggestion
      onClickOutside={closeMenu}
      onPressEscKey={closeMenu}
```